docs/sources/tempo/traceql/query-editor.md

```diff
@@ -11,20 +11,66 @@ keywords:
   - TraceQL
 ---
 
-# TraceQL query editor
+# Write TraceQL queries in Grafana
 
-You can use the TraceQL viewer and query editor in the Tempo data source to build queries and drill-down into result sets. The editor is available in Grafana’s Explore interface.
+You can compose TraceQL queries in Grafana and Grafana Cloud using **Explore** and a Tempo data source. You can use either the **Query type** > **Search** or the **TraceQL** tab.
+Both of these methods let you build queries and drill-down into result sets.
+
+To add TraceQL panels to your dashboard, refer to the [Traces panel documentation](/docs/grafana/latest/panels-visualizations/visualizations/traces/).
+
+To learn more about Grafana dashboards, refer to the [Use dashboards documentation](/docs/grafana/latest/dashboards/use-dashboards/).
 
 {{% admonition type="note" %}}
-To use the TraceQL query editor in Grafana, you need to enable the `traceqlEditor` feature flag. This feature is available starting in Grafana 9.3.2. The query editory is available automatically in Grafana Cloud.
+To use the TraceQL query editor in Grafana 9.3.2 and newer, you need to enable the `traceqlEditor` feature toggle ([refer to instructions for enabling feature toggles](/docs/grafana/latest/setup-grafana/configure-grafana/feature-toggles/).)
```

Comment:
It's not clear to me what the "TraceQL query editor" is. I think this is the search replacement with the TraceQL builder. Does this have an official name?

Reply:
Oh! Then I need to remove the feature toggle information and clarifyi the text.

Reply:
Would it be better to say the TraceQL tab instead of the query editor since we now also have a query builder in Query type > Search?